dev/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+service:
+  extends:
+    file: ../common.yml
+    service: service
+  volumes:
+    - ./payments-service/:/srv/payments-service
+  links:
+    - solitude:solitude
+
+solitude:
+  extends:
+    file: ../common.yml
+    service: service
+  volumes:
+    - ./solitude/:/srv/solitude
+  links:
+    - mysql:mysql
+
+mysql:
```

Comment:
As written here is right because `volumes_from` is one of the values not copied according to the docs. [1]

[1] http://docs.docker.com/compose/extends/#adding-and-overriding-configuration